notebooks/whisper-example.ipynb

```diff
@@ -1,354 +1,351 @@
9b8YPwffEP7iXwdPeeDagVFhmAE0kPbxU3G8ArwSqdMrEqRiMlJ90k2Cx+LbErKyYTIoYb6x9VIiwdSx8FF9wUzRHqEZkRCBGnEKcJ+AtBCcYJRQv4CCAI9ggACb4HUgrpCWMLMgp9C3kLZgrYC6EJyQh2BxoFegL7/0YCkv4e+yT2bPBg8Pbqve1C6xLlleD43C/b79o53NTUVNV6zyjMccr7zRvKW8dzyHLBXtg41tHXotGlzcrhWO4J+lru2/T87gj+PAcdEy0gUxcbEBELnB8vJ7A0Ly+SIsMj0B77JxUsADIVJn8fQBkYF5cfBCOLIwAXmBYoC3IRYRe/Ew4XSgvRDskIgw3mDQcP3xD+CfQLswmNDmwMwhBADh0PBBCUDmoPZQgYED8MwAohCiIF3AKs//EDYf6o+lP2GvI17zLs7+3n6QXpC+EP3oXcqd2X3ZfZctfMzvnT6NCA0FTRac+yzIXJmcl/0iriH90Y3KLSWtru5Cfyg/et+iP/2PVw/LgIiBW5GyIb8hW3FwYf3SHPK8ks8iqiJYwkXSXqJMEmoyO+Jr0ksyJTHgcaNhpCG0gbRR2wGggSzw/fDKURgBOSFFIRRgrlCAUHWgw7DrYPGQ5JCSIGaAcDCuEKggtUCLUGOwFR/6X+4P8QAwYAOPzN81bx5+5o8Nb07PHW7Frn398N33bjhePX5FjgHtjT1OHTtNfT2b/ZwdfS0vrOpM9b0NDWYOLC4pXhw92U26jkzOor9hb7yf/1/Kj3Lf4aBr8QLRbQFxoXmhNfFJQYaB+fJGsl2SCDH+kd+x3DH9Mn3CbBIvYjgRwKIbQddx4cHz4fhx46FzQWBhSgFWkTNxShES4R6A5eCgALTQrNDb8JCQrjBpgC1QJ8AFEBhf/RADj9tPtB+on4VPcM9Kvz5/CZ80TxGPEU7PDoFuom6SDq9Oil5afi/OGH36rhYeIQ4mveiN0K25faTt2Y3Jbe9N1F3ETgXOXg5d3qZOri6mbuMvPa+Cf/4AHb/78BKgXyB7oN6hTjFZwVsRZIFlIZVB1VI0EldiOcIzAfkyIkKNMoISqiJ38lHyF2IYkhJiBfIZgePxvKGGEV/BFREs4PNRC0DQML5AidBAoEGgKVA0kBev+O/Hj4W/g4+OP6D/sC+Qj3/PKW8FjyL/Md9WT2gPMk75btz+y57I/wM/HB7w3t8Oe65/Tpeuo86x3r5OfN5JDjOOMV5VHnt+Uu5PXh+OLj4szlpOdH6pzrUOuK7Wbvj/No9y/6GfsD/o7/ggIfBRcH0AlVCvEOzw8fE24XIRXcF7sY1hqCIHciKSH5H8EfLh7wH2wh+SB0H7scIxhnFrEVQxVCFgoSdA7AC7gI8wlZB64EKQWBAvD9Rv5r/OH62vlK+GL4C/ml+gf5tPZ/8kP0jPWR98b5kPaS8jXz9PJ/8o74RvZC+HD1GPJW7f3waPO49B/2B/I479nrVOx37KXtsfC68fXuh+1l7KDq9+zF7+PvCPAS8Bzvye2J75HwH/LQ9bT27Pqw+cr73vnV/N3/bwRiCCgI/gcIB5QIWAwXEV0UJRUsEzcRFBPyFPsVsxe/F6EUNxKdEjkSGxJCE1YOUQ/LDPkK5Qs+CKMGPQU9BT0FFwa7A0AAGwGA///8KAE8/a39qfyj+jD9OvxA/FH8GP2++t/5Pf1Y/Cj7Ifoo/mP7Qf2M/JP75frF+pX5kvoC+Ev+gfre+nP9Wfaj9tL1VPq594b5IvhZ99D4+PcA+xX3O/sW+Df55/it+kf7ufoo+6f77PtU++j9ev3j/tH+rf0qALsAXwLwAR8D4QPuAnUFHAYGB5MJMQjIBp0IjQcrCXMI+AcBC9sICgf/Bj8KPQmkCVkHLAlrBUcEYQZCBtcH8Qf5Ay0D5wXjAngGwgHtAnUCAATW/p0EGP/sAV//5AKTAi0AGAJFAMT9/P3C/6/91wB5ALf9Pv9m/Sr8jf63/lr9hP+4/F/9X/oy/pz9lvw8/Ub+PP3r/8T7H/oR/FD8Af1M/Mz+Ev4v/iH5L/6g+kv+mPsh/Rv+If2o/i//W/yG/d3+ev00/3v/e/xq/2j9ywLU/WsALv09ApX/VgGPARv+MgMnAawBcQDBAu7/UgPgAWIC1gOW/8gDLwS/AhIDxgOWAZcDAwRXAsoEigSZ/bUD/QH3A+IAewfl+hkEkv45BFYB9gCvBB4BuQCaAdj+2P7JAmr/MQGAAGwBaf1D+2wBxf8TAfgAz/2b++0A5vxQAlX9NP/2/scBDPkwAM/+xP9y/v3/hgAd/5L+P/mC/9j/8vwIBPT7Jv2q/g8AYQB8/fUB0vlOAT77KAD1/PEDCfzzBF/5JgD2+8T8MQI8APYDef7EAdH7FAFT/TQA2wFyBJQB2v//Aiv+eQH0AFT+5AWuAXT++QSK/twDmgE0AKUBtgHSAAf9NwcyAEkDFgTm/DEEKAD8/EMEFQZL/aP6bws3+pj53Apu/779dwOsAIYDcP0kBh/5ogK0A/IAafjrAQAE+v0RAZMB5AIy+rgBvvlb/5YB9QLs/jf9IQao+lL9BQRw/F8DF/42/8r7pQEiAbICS/4Z/oT8aQGYAYoETv0rAK77y/9a/a8EVf0yBJL2HwaX+QcC2gGB/QUBZv0DB3P98f87/EQG/P2e/ZYAW/+JAQgBj/94Arz9oQCP/58BwQPV/0X+DP/EAfX/LwGyA2cEgvy1/3P9xP8R/xwDUQXl/qkGvPou/Hf+8AInAUsCmv/xAWv85AbI+esC4P7R/soA5QDXAlYBQABJ/Hz5qQKgBOP5DQfgARQD//ux/2z7ZAPkBLH++PzSA8788/7++oEGbAJy/hYEgfpvAI36XQHe/oD/Kvw0AQ4Flv8HAuf7yv7C+2ECZv7KAyoBzvsIBGf4agFHBOv8c/8A+7YAzAGa/zP/ZPkQA0oAzf5SAJICjv2I/h7+pf+g/oUAzgFXANME+vsEAgD76wG2/iIFcfwtB9T3sP1qA4MADQDfAgL4Nw
```

Comment:
Add a link for the contact Graphcore. Other notebooks refer to the Graphcore Slack Community so maybe that would be suitable here.

```suggestion
    "To run these you will need more than the IPU-POD4. On Paperspace this is available using either an IPU-POD16 or a Bow Pod16 machine. Please contact Graphcore if you need assistance running these larger models.\n"
```